.gitignore

```diff
@@ -7,3 +7,7 @@ npm-debug.log
 
 # Mac OS X
 .DS_Store
+
+!intl/
+intl/*
+!intl/en/
```

Comment:
Let's not bikeshed here. It's going to work wherever you add it (and there is no convention for `.gitignores` across all LoopBack projects yet -- maybe we should come up with one and then fix them all at that time).

```
build/
reports/
!intl/
intl/*
!intl/en/
```

This part isn't alphabetized correctly in your example anyways.